hoot-core/src/main/cpp/hoot/core/conflate/DuplicateWayRemover.cpp

```diff
@@ -280,19 +290,8 @@ WayPtr DuplicateWayRemover::_getUpdatedWay(WayPtr way, const vector<long>& nodes
     WayPtr newWay;
     newWay.reset(new Way(way->getStatus(), _map->createNextWayId(), way->getRawCircularError()));
     newWay->addNodes(nodes);
-    newWay->setPid(way->getPid());
+    newWay->setPid(way->getId());
     newWay->setTags(way->getTags());
-
-    // see comments for similar functionality in HighwaySnapMerger::_mergePair
-    if (ConfigOptions().getPreserveUnknown1ElementIdWhenModifyingFeatures() &&
```

Comment:
I put in a ticket to remove all of these guys, since they don't have any purpose anymore after your recent way split changes.  I'll wait to remove them until after all the work is finished.

Reply:
Sounds good.  I've been doing them as I see them.  It is a good indication that I need to do work in that class if it is there.

Reply:
Cool.  If you need help with any of that type of stuff, let me know.